### PR TITLE
Add navigation

### PR DESCRIPTION
**Pull request by HuiminFrontend**

### CODE REVIEW

File: src/components/AddToCart/AddToCart.stories.js [hunk 1]
```diff
@@ -34,13 +35,15 @@ const currentMenu = {
...
 export default {
-  title: 'Components/AddToCart',
+  title: 'Component/AddToCart',
   component: AddToCart,
+  decorators: [
+    AddToCart => (
+      <MemoryRouter>
+        <AddToCart />
+      </MemoryRouter>
+    ),
```
Comment (macfrei) [inline src/components/AddToCart/AddToCart.stories.js:45]: I think this needs to be named `Story`:
```suggestion
   (Story) => (
      <MemoryRouter>
        <Story />
      </MemoryRouter>
    ),
```

File: src/components/AddToCart/AddToCart.js [hunk 2]
```diff
@@ -44,27 +44,22 @@ export default function AddToCart({currentMenu}) {
...
   width: 100%;
   height: 3rem;
-  position: sticky;
-  bottom: 0;
-  left: 0;
   font-weight: 700;
-  display: flex;
-  justify-content: space-evenly;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```
Comment (JessicaLoers) [inline src/components/AddToCart/AddToCart.js]: A very picky note and as a small repetition. You could also write it like this 👇
```suggestion
  grid-template-columns: repeat(2, 1fr);
```

File: src/components/BackToPreviousPage/BackToPreviousPage.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import {MdOutlineArrowBackIosNew} from 'react-icons/md';
+import {useNavigate} from 'react-router-dom';
+import styled from 'styled-components';
+
+export default function BackToPreviousPage() {
+  const navigate = useNavigate();
+  return (
+    <GoBack onClick={() => navigate(-1)}>
```
Comment (JessicaLoers) [inline src/components/BackToPreviousPage/BackToPreviousPage.js:8]: If you want to redirect to as specific page, you could use navigate() to call a path directly, for example to the start page
```suggestion
    <GoBack onClick={() => navigate("/")}>
```

File: src/components/Navigation/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+import {MdMenuBook} from 'react-icons/md';
+import {NavLink} from 'react-router-dom';
+import styled from 'styled-components';
+
+export default function Navigation() {
+  return (
+    <Nav>
+      <StyledUl>
+        <li>
+          <NavLink to="/" style={({isActive}) => (isActive ? {color: '#f77c00'} : {color: 'black'})}>
```
Comment (JessicaLoers) [inline src/components/Navigation/Navigation.js]: I highly recommend to not use inline styles. 
Rather create a styled component with `props` for your `<NavLink>` 
```suggestion
         const StyledNavLink = styled(NavLink) `
           ...... css rules here
         `
```

File: src/components/Navigation/Navigation.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            <NavContainer>
+              <MdMenuBook data-testid="menus-icon" />
+              <Span>Menus</Span>
+            </NavContainer>
+          </NavLink>
+        </li>
+        <li>
+          <NavLink to="/cart" style={({isActive}) => (isActive ? {color: '#f77c00'} : {color: 'black'})}>
```
Comment (JessicaLoers) [inline src/components/Navigation/Navigation.js]: Same here 🙂

**Review by JessicaLoers — COMMENTED** [review]
Hi @Huimin66 
A really cool App - I get hungry right away 🙂

I have a few minor suggestions and comments.

File: src/components/BackToPreviousPage/BackToPreviousPage.js (hunk 3 above)
Comment (HuiminFrontend) [reply src/components/BackToPreviousPage/BackToPreviousPage.js:8]: Thank you,  good to know. I just need to go to previous page.

**Pull request merged**